lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -713,6 +713,7 @@ class KymotrackerParameter:
     lower_bound: Number
     upper_bound: Number
     ui_visible: bool
+    info: str
```

Comment:
I would prefer `extended_description` so the relationship between the two fields is made clear.

Reply:
Fixed!